### ISSUE TITLE
Power Settings affect starting of Scheduled Task

### ISSUE THREAD

**Issue by JoeTalbot** [smurawski/chef-zero-scheduled-task#11]
Using a mac, the scheduled task fails to start and hangs at this part of the chef run:
```
-----> Chef Omnibus installation detected (install only if missing)
       Creating the scheduled task.
       SUCCESS: The scheduled task "chef-tk" has successfully been created.
       Transferring files to <windows-windows-2012r2-vagrant>
       SUCCESS: Attempted to run the scheduled task "chef-tk”.
```
I am using mac osx, and i found it is due to running on battery, and is passing the battery state to the virualbox vm that is being spun up with test-kitchen, and the setting “Start the task only if the computer is on AC power”

**Comment by stevenao** [comment]
Starting to see the same problem as well on host os windows 8.1!

**Comment by smurawski** [comment]
I've released v0.2.0 of this gem which deprecates its use and suggests the use of the `elevated` configuration on the transport.
